backend/package.json

```diff
@@ -21,6 +21,8 @@
 		"moment": "^2.29.4",
 		"mysql2": "^3.11.1",
 		"node-rsa": "^1.0.8",
+		"nodemon": "^2.0.2",
```

Comment:
`nodemon` is already in `devDependencies` - adding it here is unnecessary and will add bloat to the final image